packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -70,11 +70,14 @@ gmaps.MapOptions _configurationAndStyleToGmapsOptions(
     options.zoomControl = configuration.zoomControlsEnabled;
   }
 
-  if (configuration.scrollGesturesEnabled == false ||
-      configuration.zoomGesturesEnabled == false) {
```

Comment:
I think that if the user is setting `scrollGesturesEnabled` or `zoomGesturesEnabled` to false, we should at least warn them that they now need to set `webGestureHandling` to some of its values instead (like "none"). Otherwise, we're just setting gestureHandling to `auto` when they possibly wanted `none`.

Reply:
How should we warn them?

Reply:
Through the js `console` when in debug mode, like here:

* https://github.com/flutter/packages/blob/4c1149744dbb36984ec5e19a61443d91b7ecefb6/packages/google_sign_in/google_sign_in_web/lib/google_sign_in_web.dart#L222-L227

google_maps_flutter_web is using `dart:html` already, so you should be able to use this JS console object:

* https://api.flutter.dev/flutter/dart-html/Console/warn.html (`window.console.warn`)